fang/Cargo.toml

```diff
@@ -51,8 +102,29 @@ thiserror = "1.0"
 typed-builder = "0.14"
 typetag = "0.2"
 uuid = { version = "1.1", features = ["v4"] }
-fang-derive-error = { version = "0.1.0" , optional = true}
-sqlx = {version = "0.6.3", features = ["any" , "macros" , "chrono", "uuid", "json", "runtime-tokio-rustls"], optional = true}
+fang-derive-error = { version = "0.1.0", optional = true }
+#sqlx = { version = "0.8", features = [
+#    "any",
+#    "macros",
+#    "chrono",
+#    "uuid",
+#    "json",
+#    "runtime-tokio-rustls",
+#    "postgres",
+#    "sqlite",
+#    "mysql",
+#] }
+sqlx = { git = "https://github.com/launchbadge/sqlx.git", branch = "main", features = [
```

Comment:
it won't be possible to release a new version if the crate will reference a github repo

Reply:
I know , I am officially waiting for the sqlx 8.3 release.